projects/Mallard/package.json

```diff
@@ -4,8 +4,8 @@
     "private": true,
     "scripts": {
         "pre-run": "yarn watch-html & node ./scripts/versioning.js",
-        "pre-bundle": "yarn bundle-html && node ./scripts/versioning.js",
-        "postinstall": "yarn pre-bundle",
+        "pre-bundle": "yarn bundle-html",
+        "postinstall": "node ./scripts/versioning.js",
```

Comment:
With `bundle-html` in the postinstall there's a race condition when the initial install is run as it could call the build on the crosswords directory while it's installing.